src/fides/api/util/consent_util.py

```diff
@@ -228,3 +247,278 @@ def get_or_create_fides_user_device_id_provided_identity(
         )
 
     return identity  # type: ignore[return-value]
+
+
+def validate_notice_data_uses(
+    privacy_notices: List[Union[PrivacyNoticeWithId, PrivacyNoticeCreation]],
```

Comment:
This method was just moved from src/fides/api/api/v1/endpoints/privacy_notice_endpoints.py.  The only change is the type signature so it can be shared to validate data uses before creating a Notice or a Template